README.md

```diff
@@ -7,19 +17,22 @@
 Floras documentation can be found [here](https://floras.readthedocs.io).
 
 ### Requirements
-Floras requires `Python>=3.10` and a C++17-compliant compiler (for example `g++>=7.0` or `clang++>=5.0`).
+Floras requires `Python>=3.10,<3.13` and a C++17-compliant compiler (for example `g++>=7.0` or `clang++>=5.0`).
 You can check the versions by running `python --version` and `gcc --version`.
-#### MacOS
-If you are using a Mac, please pre-install [graphviz](https://graphviz.org) and [pygraphviz](https://pygraphviz.github.io).
-Using [conda](https://conda.org/):
+
+#### Pre-installing Graphviz
+Please pre-install [graphviz](https://graphviz.org) and [pygraphviz](https://pygraphviz.github.io).
+If you are using a Mac, please install it via [brew](https://brew.sh) and [pip](https://pypi.org/project/pip/):
 ```
-conda install --channel conda-forge pygraphviz
+brew install graphviz
+pip install pygraphviz
```

Comment:
This command fails for me on macOS 15.2. I had to use the corresponding command from macos_build_pdm.yaml:

```sh
pip install --no-cache-dir \
    --config-settings="--global-option=build_ext" \
    --config-settings="--global-option=-I$(brew --prefix graphviz)/include/" \
    --config-settings="--global-option=-L$(brew --prefix graphviz)/lib/" \
    pygraphviz
```
To keep the README concise, I suggest that special cases like this are handled in docs/installing.md and that the README points at docs/installing.md. Perhaps some text like "if these instructions do not work for you, please find more detail and alternatives in docs/installing.md."